### PR TITLE
add Python_zh_cn

### PR DESCRIPTION
**Pull request by bokix**
add python chinese API document, include python2.7.16 and python3.7.4

### CODE REVIEW

File: docsets/Python_zh_cn/docset.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "name": "Python_zh_cn",
+    "version": "3.7.4",
+    "archive": "Python_zh_cn.tgz",
```
Comment (Kapeli) [inline docsets/Python_zh_cn/docset.json]: You need to add this archive as well.

File: docsets/Python_zh_cn/docset.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+    "name": "Python_zh_cn",
```
Comment (Kapeli) [inline docsets/Python_zh_cn/docset.json]: Please set the name to "Python (Chinese)"

File: docsets/Python_zh_cn/docset.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+                "cn",
+                "zh"],
+
+    "specific_versions": [
+        {
+            "_comment": "API document for python2 and python3"
+        },
+        { 
+            "version": "3.7.4",
+            "archive": "versions/3.7.4/Python_zh_cn.tgz",
```
Comment (Kapeli) [inline docsets/Python_zh_cn/docset.json]: This path does not exist. The path you have is: `versions/3.7.4.1/Python_zh_cn.tgz`.

Comment (bokix) [reply docsets/Python_zh_cn/docset.json]: Sorry! but in Mac Finder, the folder name is "3.7.4" and in Terminal is "3.7.4.1" , I don't know why. Anyway, I modified the name of the folder.

**Comment by Kapeli** [comment]
Thank you! 👍

**Pull request merged**